hack/Dockerfile.python-client

```diff
@@ -0,0 +1,5 @@
+FROM quay.io/edge-infrastructure/swagger-codegen-cli:2.4.18
```

Comment:
Could you rename this file to `Dockerfile.python-client-generate`? Current name is a bit misleading